cluster/plan.go

```diff
@@ -174,7 +174,28 @@ func (c *Cluster) BuildKubeAPIProcess(prefixPath string) v3.Process {
 		services.SidekickContainerName,
 	}
 	Binds := []string{
-		fmt.Sprintf("%s:/etc/kubernetes:z", path.Join(prefixPath, "/etc/kubernetes")),
+		fmt.Sprintf("%s:%s:ro", path.Join(prefixPath, pki.GetCertPath(pki.CACertName)), pki.GetCertPath(pki.CACertName)),
```

Comment:
can you explain how this will improve the security of managing certificates, this solution will also break the ability to use custom certificates deployed in /etc/kubernetes